releases/release-1.31/release-notes/maps/pr-123905-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 123905
+releasenote:
+  text: |-
```

Comment:
what is this '|-' , I don't think it's necessary but @npolshakova would you help me confirm it ? thanks :)

Reply:
Generally |- is used for multiline yaml without line break at the end but i don't think its required here

Reply:
Yep, `|-` is used for multiline yaml. If it's not required, feel free to remove it. In general we just want to make sure the yaml is valid and correctly formatted. If a line is too long, we should use the multiline yaml formatting to make it easier to review.